CMakeLists.txt

```diff
@@ -155,6 +158,12 @@ if (ENABLE_DISCORD_RPC)
 	target_include_directories(discord-rpc INTERFACE ./dependencies/discord-rpc/include)
 endif()
 
+if (ENABLE_HIDAPI)
+	find_package(hidapi REQUIRED)
+	set(ENABLE_WIIMOTE ON)
+	add_compile_definitions(HAS_HIDAPI)
```

Comment:
Could you please move this to `src/input/CMakeLists.txt` and use `target_compile_definitions(PRIVATE)` instead? It'd be a bit cleaner, in my opinion.

Reply:
In that case consistency is better than supposed code clarity, it's fine to leave this as is :)